server/build.go

```diff
@@ -197,6 +197,8 @@ func (task *BuildTask) build(marker *stringSet) (esm *ESM, err error) {
 		buf := bytes.NewBuffer(nil)
 		importPath := task.Pkg.ImportPath()
 		fmt.Fprintf(buf, `import * as __module from "%s";`, importPath)
+		// Default reexport all members from original module to prevent missing named exports members
+		fmt.Fprintf(buf, `export * from "%s";`, importPath)
```

Comment:
maybe just add this when `len(esm.Exports) === 0` ?

Reply:
I think can always export from the origin, and the `esm.Exports` will overwrite the same  name exports